creation/server/config.lua

```diff
@@ -0,0 +1,56 @@
+Config = Config or {}
+Config.ConfigFormat = false
+-- Default Format
+
+-- Name: TestBox | 2022-04-13T22:46:17Z
+-- BoxZone:Create(vector3(-344.16, -103.25, 39.02), 1, 1, {
+--     name="TestBox",
+--     heading=0,
+--     --debugPoly=true
```

Comment:
Can you please add spaces around the `=` in all the places there aren't any in this file?